packages/hub/node-tests/cards-test.js

```diff
@@ -1,165 +1,133 @@
-
+const JSONAPIFactory = require('@cardstack/test-support/jsonapi-factory');
+const { addCardContext } = require('@cardstack/test-support/card-helpers');
 const {
   createDefaultEnvironment,
   destroyDefaultEnvironment
 } = require('../../../tests/stub-searcher/node_modules/@cardstack/test-support/env');
 
+// addCardContext() is a test helper that imbues the compound ID's into the card document
+// based on the repository and module name of the card. This allows our tests to create card documents
+// using the same API that developers would use for creating card documents with the
+// JSONAPIFactory within the respective card modules that a card document originates from.
+let articleCard = addCardContext('local-hub', 'article-card',
```

Comment:
@ef4 here's an example of adding context to a card using a test helper so that the card authoring experience in our tests can mimic the authoring experience that developers would have when writing a card document.

Reply:
There should not be two different forms of a card's json:api document. 

There will be an on-disk format for cards, but that is an internal concern of the indexer/searcher that knows how to read & write the on-disk format. The entire rest of the system only knows about the json:api format.

If it turns out nice to write tests using the on-disk format, then the tests should use that indexer/searcher.

Reply:
Also, the schema in the on-disk format should be a json file, and the normal way to edit it will be to do it in the UI and let the writer update the json file. Focus on that use case. Up until now we have been doing everything by hand, even though we know that's not the experience we're going for. While it will always be possible to hand-edit these files, we want to focus primarily on the happy path where people are editing schema in the UI.